playbooks/configure-dns-with-router-ip.yml

```diff
@@ -0,0 +1,66 @@
+# Configure the dns of the masters and nodes 
+# with the ip and and subdomain of the cluster's router.
+# After runnin this playbook on a master/node, openshift routes
```

Comment:
s/runnin/running
s/a master\/node/master and node machines/
s/openshift/OpenShift